test/correctness/async_device_copy.cpp

```diff
@@ -0,0 +1,91 @@
+#include "Halide.h"
+#include "halide_benchmark.h"
+
+using namespace Halide;
+
+// Return zero, slowly
+Expr expensive_zero(Expr x, Expr y, Expr t, int n) {
+    // Count how many Fermat's last theorem counterexamples we can find using n trials.
```

Comment:
I have discovered a truly marvelous proof of this, which this column-width is too narrow to contain